icu4j/main/classes/core/src/com/ibm/icu/text/DateTimePatternGenerator.java

```diff
@@ -1205,6 +1205,35 @@ private String cldrKey() {
     private static final int APPENDITEM_WIDTH_INT = APPENDITEM_WIDTH.ordinal();
     private static final DisplayWidth[] CLDR_FIELD_WIDTH = DisplayWidth.values();
 
+    /**
+     * Hour Cycle
+     * @draft ICU 67
+     */
+    public enum HourCycle {
+        /**
+         * hour in am/pm (1~11)
+         * @draft ICU 67
+         */
+        HourCycle11,
```

Comment:
You should use UPPER_CONSTANT_CASE for enums

Reply:
DONE